src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/UserManagementResource.kt

```diff
@@ -35,6 +35,49 @@ class UserManagementResource(
   private val userService: UserService,
 ) {
 
+  @PreAuthorize("hasRole('ROLE_MANAGE_NOMIS_USER_ACCOUNT')")
+  @PutMapping("/{username}/logon-date")
```

Comment:
Couple of points:
1. Last time I worked on HMPPS Auth we had to call it a sign-in, not a logon.
2. Seems slightly odd to PUT to `/users/{username}/logon-date`.  Since you're not passing in the date then it feels more like a `POST`.  Also, as you've mentioned in summary, the important bit is to record the information - so would suggest a POST to `/users/{username}/record-sign-in`

Reply:
Thanks Pete, ideally the new field and trigger would be using 'sign-in' rather than 'logon' so I was trying to keep the naming consistent.  But I guess I have the opportunity at this API interface to correct the terminology. I'll change it 👍 

Also agree with the PUT -> POST, will change that too.

Reply:
Done